### PR TITLE
NH-5414-Update-Dependencies-devDependencies-2

### PR DESCRIPTION
**Pull request by ronilan**
### Overview:

This pull request removes `methods` from dependencies.

`methods` was added as dependency in https://github.com/appoptics/appoptics-apm-node/commit/a04fafd78ea8a8d9e8ef1a6672a634c02b6b35e2

### Status:

Package [does little](https://www.npmjs.com/package/methods) and can be replaced with native `http.METHODS`.

### Notes:
- Pull Request merges into `nh-main`. Agent built from `master` will maintain dependency`.

### CODE REVIEW

File: lib/probes/express.js [hunk 1]
```diff
@@ -128,13 +129,13 @@ function patchRoutes (express, version) {
       logMissing('v4 Route.prototype')
       return
     }
-    methods.concat('all').forEach(method => {
-      if (typeof proto[method] !== 'function') {
-        logMissing(`v4 Route.${method}()`)
+    http.METHODS.concat('all').forEach(method => {
+      if (typeof proto[method.toLowerCase()] !== 'function') {
+        logMissing(`v4 Route.${method.toLowerCase().to}()`)
```
Comment (cheempz) [inline lib/probes/express.js:134]: In `method.toLowerCase().to` what does the `.to` do?

**Review by cheempz — COMMENTED** [review]
LGTM pending one comment.

File: lib/probes/express.js (hunk 1 above)
Comment (ronilan) [reply lib/probes/express.js]: It was s a typo that passed tests. Fixed. Thanks.

**Review by cheempz — APPROVED** [review]
LGTM, thanks @ronilan

**Pull request merged**